### PR TITLE
Added Social Buttons to the footer in place of the floating button.

### PR DESCRIPTION
**Pull request by arpitkamboj**
GCI Task 2017

Changed the location of social media buttons as requested in the task.

Please review changes.

### CODE REVIEW

**Comment by smsunarto** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/104 :rocket:

**Comment by arpitkamboj** [comment]
Hmm...Why can't I see the changes?

Well.. If this does not work then please check the changes on http://winbox.phenylinux.com:4000/

**Comment by smsunarto** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/104 :rocket:

**Comment by arpitkamboj** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/104 :rocket:

**Comment by arpitkamboj** [comment]
I made the suggested change and I think it's ready for review now...
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/104 :rocket:

**Comment by arpitkamboj** [comment]
Fixed the color of the License text. Looks better now.

**Comment by arpitkamboj** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/104 :rocket:

**Comment by arpitkamboj** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/104 :rocket:

File: _includes/footer.html [hunk 1]
```diff
@@ -1,9 +1,22 @@
 <!-- Start ignoring HTMLLintBear -->
-<footer class="page-footer">
-  <div class="footer-copyright light-green darken-3">
-    <div class="container light-green-text">
-      <strong>Licensed under <a class="green-text text-darken-4" href="https://www.apache.org/licenses/LICENSE-2.0">Apache 2.0</a></strong>
-    </div>
+<footer class="page-footer light-green">
+  <div class="container light-green">
```
Comment (smsunarto) [inline _includes/footer.html]: You don't need to declare light-green here again.

File: _includes/footer.html [hunk 2]
```diff
@@ -1,9 +1,22 @@
 <!-- Start ignoring HTMLLintBear -->
-<footer class="page-footer">
-  <div class="footer-copyright light-green darken-3">
-    <div class="container light-green-text">
-      <strong>Licensed under <a class="green-text text-darken-4" href="https://www.apache.org/licenses/LICENSE-2.0">Apache 2.0</a></strong>
-    </div>
+<footer class="page-footer light-green">
```
Comment (smsunarto) [inline _includes/footer.html]: I think I prefer `light-green darken-2` here

**Review by smsunarto — COMMENTED** [review]
Good job! Small reviews here.

File: _includes/footer.html (hunk 2 above)
Comment (arpitkamboj) [reply _includes/footer.html]: OK

File: _includes/footer.html (hunk 1 above)
Comment (arpitkamboj) [reply _includes/footer.html]: Forgot...

**Comment by arpitkamboj** [comment]
Should be fine now...

Check this out
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/104 :rocket:

**Comment by arpitkamboj** [comment]
Let's preview this

**Comment by gooey-splashsite** [comment]
Preview is available on http://preview.terasology.net/splashsite/pull/104 :rocket:

**Comment by smsunarto** [comment]
Thanks! Approving the task.

**Comment by arpitkamboj** [comment]
Thank you!

**Pull request merged**